tests/ui/traits/new-solver/specialization/specialization-default-no-solution.rs

```diff
@@ -0,0 +1,26 @@
+// check-pass
```

Comment:
want a second test:

`normalizes-to` is `NoSolution` if there is no specializing impl, projection remains rigid

`normalizes-to` chooses the specializing impl over the default impl where applicable